SAASINT-4828 Update Hubspot public docs with reference table information (#21790)
DataDog/integrations-core: hubspot_content_hub/README.md

3. To add an account to install the integration, click the **Add HubSpot Account** button.
4. After reading the instructions in the modal, click the **Authorize** button, which redirects you to the HubSpot login page.
5. After logging in, you are prompted to select which HubSpot account you want to grant access to.
6. Click **Authorize**.

> 7. You're redirected back to Datadog's HubSpot tile with a new account. Datadog recommends changing the account name to something that is easier to remember. You can add multiple accounts with access to different organizations.

Reviewer: ```suggestion
7. You’re redirected to Datadog’s HubSpot tile with a new account. Datadog recommends renaming the account to make it more recognizable. You can add multiple accounts, each with access to different organizations.
```

> If the **Create Companies Reference Table** toggle is enabled, a [Datadog Reference Table][5] containing data on companies from your Hubspot account will be created. This requires the **crm.objects.companies.read** permission on Hubspot.

Reviewer: ```suggestion
If the **Create Companies Reference Table** toggle is enabled, Datadog creates a [Reference Table][5] containing company data from your HubSpot account. This action requires the **crm.objects.companies.read** permission in Hubspot.
```
